app/assets/stylesheets/searchworks4/modal.css

```diff
@@ -10,3 +10,71 @@
     --bs-modal-padding: 1rem;
   }
 }
+
+/* The feedback form is available in both modal and standalone page form. The parent classes below cover both cases */
+.modal:has(.feedback-form), div[data-blacklight-modal] {
```

Comment:
Where in the HTML do we see `div[data-blacklight-modal]`?

Reply:
It is here: https://github.com/projectblacklight/blacklight/blob/0668e8e8466d00d2db32843ccfd15aae57ec65e8/app/components/blacklight/system/modal_component.html.erb#L1